.github/workflows/deploy-gcp-tests.yml

```diff
@@ -566,7 +605,6 @@ jobs:
       - name: Show logs for ${{ inputs.test_id }} test (sprout)
         run: |
           gcloud compute ssh ${{ inputs.test_id }}-${{ env.GITHUB_REF_SLUG_URL }}-${{ env.GITHUB_SHA_SHORT }} \
-          --ssh-key-expire-after=30s \
```

Comment:
This job does not install the key, but it looks like it should.

About half the jobs in this file are missing an install step.

Reply:
Solved with [fix: add missing SSH key installation steps](https://github.com/ZcashFoundation/zebra/pull/5671/commits/81c74a292f6f936f4ec574bf40688aca9dae4be0)